xtask/src/ci/spec_links.rs

```diff
@@ -216,7 +216,8 @@ fn get_page_ids(url: &str) -> Result<HashMap<String, HasDuplicates>> {
     let html = page.text()?;
     let mut ids = HashMap::new();
 
-    for token in Tokenizer::new(&html).infallible() {
+    for token in Tokenizer::new(&html) {
+        let Ok(token) = token;
```

Comment:
Any reason not to do `for Ok(token) in ...`?

Reply:
None at all. Thanks for flagging!